aim/web/ui/src/services/models/runs/util.ts

```diff
@@ -47,6 +48,16 @@ export function getMenuItemFromRawInfo(
 
   const data: IMenuItem[] = [];
 
+  const sortOrder = alphabeticalSortComparator({
+    orderBy: 'name',
+    additionalCompare: (name1: string, name2: string) => {
+      if (name2 === 'EMPTY CONTEXT') {
+        return 0;
```

Comment:
0 or -1 ?

Reply:
It need to show at the beginning of the list.
In the menu list of TraceVisualizer if there is an empty context, it is always positioned at the beginning of the list, so it need to keep position the same.

Please also see the comment in line 108, 109 of this file